agent/js/src/wd_server.js

```diff
@@ -155,7 +155,8 @@ WebDriverServer.prototype.initIpc = function() {
  */
 WebDriverServer.prototype.init = function(args) {
   'use strict';
-  args.startURL = args.task['Capture Video'] ? ORANGE_PAGE_URL_ : BLANK_PAGE_URL_;
+  args.startURL = args.task['Capture Video'] && !args.task.lighthouse ? 
+      ORANGE_PAGE_URL_ : BLANK_PAGE_URL_;

```

Comment:
I think it'll be better to have lighthouse change it's about:blank to arbitrary this rather than try to orchestrate the tab ID navigation, etc